### PR TITLE
Adding a -q option to the logstart magic to suppress output of log state

### LINKED ISSUE #3398: Option to disable printing logstate on log activation
/usr/lib/python2.7/site-packages/IPython/core/magics/logging.py Line 135

printing the log state everytime shell is opened is annoying, please make it configurable.

### PR DESCRIPTION
**Pull request by joshuastorck**
Also added a command line option. Resolves #3398.

### CODE REVIEW

File: IPython/core/tests/test_magic.py [hunk 1]
```diff
@@ -977,3 +978,24 @@ def sii(s):
...
+    ip.run_line_magic('logstop', '')
+    ip.config.LoggingMagics.quiet = True
+    lm = logging.LoggingMagics(shell=ip)
+    with tt.AssertNotPrints(re.compile("Activating.*")):
+        lm.logstart()
+    
+def test_logging_magic_not_quiet():
+    ip = get_ipython()
+    ip.run_line_magic('logstop', '')
+    with tt.AssertPrints(re.compile("Activating.*")):
```
Comment (takluyver) [inline IPython/core/tests/test_magic.py]: This test seems to be failing on Travis - can you take a look?

**Comment by joshuastorck** [comment]
@Carreau, I'm getting errors creating temp logs on Windows. Any suggestions on how to fix this and/or what might be going wrong?

**Comment by takluyver** [comment]
Ugh, that's a bit messy. You need to stop the logging before the temporary directory is cleaned up - Windows can't remove the log file while it's still open.

**Comment by takluyver** [comment]
('messy' refers to the general cleanup situation, btw, not your code ;-)

**Comment by joshuastorck** [comment]
Ok, fixed the Windows problem. Before merging this request, what do folks think about making the default quiet and instead have %logstart -v or LoggingMagics.verbose=True?

**Comment by takluyver** [comment]
I ~never use the logging feature, so my opinion isn't that important, but as you can just type `%logstart` and it picks a set of default options, I think it's good for it to show what those options are, without needing any extra flag.

**Comment by joshuastorck** [comment]
@takluyver, PR is ready for merge.

**Comment by Carreau** [comment]
Thanks perfect ! Merging  ! If you want you can make another PR drop a file in `docs/source/whatsnew/pr/` which describe the new feature (and add a "Contributed by" if you like). It would be merged into the release notes at release time !

Thanks !

**Pull request merged**